examples/docker-build/.gitignore

```diff
@@ -0,0 +1,84 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
To keep example small, I suggest removing extra boilerplate like `.gitignore` from the example.